channeldb/mp_payment.go

```diff
@@ -18,6 +22,9 @@ type HTLCWireInfo struct {
 
 	// Route is the route attempted to send the HTLC.
 	Route route.Route
+
+	// AttemptTime is the time at which this HTLC was attempted.
+	AttemptTime time.Time
```

Comment:
why does this move? it isn't wire info

Reply:
Commit reverted, it is now an addition an not a move anymore.